src/metatrain/__init__.py

```diff
@@ -1,11 +1,11 @@
 from pathlib import Path
 import secrets
 
+from ._version import __version__, __version_tuple__  # noqa
```

Comment:
```suggestion
from ._version import __version__, __version_tuple__  # noqa: F401
```

I'm also not sure we need `__version_tuple__` here